benchexec/tools/ultimate.py

```diff
@@ -66,6 +66,9 @@ class UltimateTool(benchexec.tools.template.BaseTool2):
     def __init__(self):
         self.java = None
 
+    def project_url(self):
+        return "https://github.com/ultimate-pa/ultimate"
```

Comment:
This URL should be https://www.ultimate-pa.org

Reply:
Changed it, but then it is the wrong url in sys-lab:main, too